src/twisted/internet/abstract.py

```diff
@@ -481,30 +481,49 @@ def fileno(self):
         return -1
 
 
-def isIPAddress(addr):
+
+def isIPAddress(addr, family=AF_INET):
     """
-    Determine whether the given string represents an IPv4 address.
+    Determine whether the given string represents an IP address of the given
+    family; by default, an IPv4 address.
 
     @type addr: C{str}
     @param addr: A string which may or may not be the decimal dotted
-    representation of an IPv4 address.
+        representation of an IPv4 address.
+
+    @param family: The address family to test for.  (This parameter has only
+        been available since Twisted 16.7; previosuly L{isIPAddress} could only
+        test for IPv4 addresses.)
+    @type family: C{bool}
```

Comment:
It seems like `bool` isn't the correct type for the address family parameter. Would `C{int}` or something more descriptive like what is found in [twisted/tcp.py](https://github.com/twisted/twisted/blob/e38cc25a67747899c6984d6ebaa8d3d134799415/src/twisted/internet/tcp.py#L329-L332) be more appropriate?